core/templates/dev/head/pages/exploration_editor/editor_tab/StateSolution.js

```diff
@@ -147,7 +145,7 @@ oppia.controller('StateSolution', [
               currentStateName, false);
             explorationWarningsService.updateWarnings();
             alertsService.addInfoMessage(
-              'That solution does not lead to the next state!');
+              'Current solution does not lead to another card.');
```

Comment:
Here and below: pull these constants out into e.g. INFO_MESSAGE_SOLUTION_IS_VALID, INFO_MESSAGE_SOLUTION_IS_INVALID if you're going to use them in multiple places throughout the app.